src/mlnet/Commands/New/NewCommandHandler.cs

```diff
@@ -21,7 +21,7 @@ internal NewCommand(NewCommandSettings settings, MlTelemetry telemetry)
 
         public void Execute()
         {
-            telemetry.LogAutoTrainMlCommand(settings.Dataset.FullName, settings.MlTask.ToString(), settings.Dataset.Length);
+            telemetry.LogAutoTrainMlCommand(settings.Dataset.Name, settings.MlTask.ToString(), settings.Dataset.Length);
```

Comment:
What is FullName & Name? Is this the filepath & filename?

Reply:
FullName would have both name and path. We log this mostly to try to tell how much use is from samples, folks will have different folders but same files so Name is what we really want.